### ISSUE TITLE
Swap two numbers without third variable

### ISSUE THREAD

**Issue by niranjantuty** [sukritishah15/DS-Algo-Point#593]
## 🚀 Feature

Given two variables x and y, swap two variables without using a third variable.

Input:
x = 2, y = 1
Output:
x = 1, y = 2

### Have you read the Contribution Guidelines?

Yes.

## Pitch

This is a common interview question asked for freshers.

**Comment by RohanK6** [comment]
If available, I can do this in Java.

**Comment by ankit9905** [comment]
i Can do this in C++

**Comment by DefCon-018** [comment]
i want to do this in python

**Comment by yashkansal** [comment]
I can do this in c

**Comment by Shrey2002** [comment]
i can do this in C#

**Comment by niranjantuty** [comment]
@RohanK6 - Java
@ankit9905 - C++
@DefCon-018 - Python
@yashkansal - C
@Shrey2002  - C#

Assigned.  Go ahead :)

**Comment by Shrey2002** [comment]
Can I also get this in PHP?

**Comment by arsh9806** [comment]
Please assign this to me for Javascript

**Comment by niranjantuty** [comment]
@Shrey2002  - PHP 
@arsh9806 - JS
Assigned !

**Comment by DineshSolanki** [comment]
please can you assign me this for Kotlin, I'll be glad to do it in Kotlin.

**Comment by IshaanSinghal1** [comment]
Assign me to do it in Kotlin.

**Comment by niranjantuty** [comment]
@DineshSolanki  - Kotlin Assigned (fcfs)

**Comment by beardedasianlad** [comment]
I would like to tackle the golang version of this.

**Comment by niranjantuty** [comment]
@beardedasianlad  - GO 
Assigned!

**Comment by kesher988** [comment]
can i do it in c

**Comment by Aditya489** [comment]
can you assign me this task ? thank you

**Comment by Aditya489** [comment]
> can you assign me this task ? thank you

i can do this in python

**Comment by richard-oden** [comment]
May I try this in Ruby?

**Comment by chandrikagottappu** [comment]
can i do this in ruby?

**Comment by chandrikagottappu** [comment]
can i do this task?

**Comment by sudsali** [comment]
please assign this to me i can do this in c++

**Comment by AvadhootK** [comment]
please assign this to me for Java.

**Comment by MERCUROUS** [comment]
please assign this to me I can do this in C, Java, Python

**Comment by SpaceBarDoodle** [comment]
Please assign me this issue i can do this in bash script.And please add label of bash script and hacktoberfest.

**Comment by FionaFR** [comment]
Can you please assign  Scala for me?

**Comment by niranjantuty** [comment]
@FionaFR  - Scala 
assigned!

**Comment by FionaFR** [comment]
Can you please add #hacktoberfest-accepted for this issue?

**Comment by niranjantuty** [comment]
@Shrey2002  You're still working on this in PHP?

**Comment by va1m** [comment]
```java
var a = 5;
var b = 7;

a ^= b;
b ^= a;
a ^= b;

assert a == 7;
assert b == 5;
```

**Comment by HitIt007** [comment]
I want to this java, python,c, c++

**Comment by va1m** [comment]
> I want to this java, python,c, c++

This is java: https://github.com/sukritishah15/DS-Algo-Point/issues/593#issuecomment-708530361

**Comment by LaxmiNyamagoud** [comment]
can I work on php?

**Comment by amankeshari** [comment]
**I want to do this in C**

**Comment by sunilgknair051** [comment]
A fair warning to all the newcomers commenting here and opening PRs without being assigned. Make sure you read the contributing.md before you ahead do anything. In the end, if you haven't been assigned, your PR gets rejected.

**Comment by LaxmiNyamagoud** [comment]
> A fair warning to all the newcomers commenting here and opening PRs without being assigned. Make sure you read the contributing.md before you ahead do anything. In the end, if you haven't been assigned, your PR gets rejected.

I am really sorry, I sent above program by mistake.

**Comment by sunilgknair051** [comment]
@LaxmiNyamagoud, don't worry, go through the issues, find the one you can solve, and get yourself assigned.

**Comment by LaxmiNyamagoud** [comment]
ok

**Comment by LaxmiNyamagoud** [comment]
> can I work on php?

please assign this issue to me in php

**Comment by sunilgknair051** [comment]
@LaxmiNyamagoud , please go through the issue from the start, and look for a word called **Assigned**, if a person hasn't been assigned to implement it in PHP, then I will assign it to you. Could you please tell me if it has been assigned to someone?

**Comment by LaxmiNyamagoud** [comment]
> @LaxmiNyamagoud , please go through the issue from the start, and look for a word called **Assigned**, if a person hasn't been assigned to implement it in PHP, then I will assign it to you. Could you please tell me if it has been assigned to someone?

assigned!

**Comment by LaxmiNyamagoud** [comment]
> @LaxmiNyamagoud , please go through the issue from the start, and look for a word called **Assigned**, if a person hasn't been assigned to implement it in PHP, then I will assign it to you. Could you please tell me if it has been assigned to someone?

assigned!

**Comment by sunilgknair051** [comment]
yep, so please choose another language to implement or another issue.

**Comment by TusharKalyankar** [comment]
@niranjantuty  I CAN  DO THIS USING JAVA

**Comment by rakshitgupta23** [comment]
@sukritishah15 @niranjantuty Please assign it to mr in java or C.

**Comment by sukritishah15** [comment]
@rakshitgupta23 - This has already been completed in Java and C.

P.S. I am the sole maintainer of this repository at this point of time, and no one else would/should respond regarding issue assignment or PR approval. Please respect past maintainers' time!

Thank you!

**Comment by its-kundan** [comment]
assign it to me 
I can do it in kotlin

**Comment by its-kundan** [comment]
assign me 
I can do it in kotlin

**Comment by shubhig226** [comment]
@niranjantuty I can do this in java. Please assign this to me.

**Comment by Adithya4720** [comment]
I can do this in Rust.Can U assign it to me

**Comment by Rishabh-mikku** [comment]
Hey @niranjantuty I can do this in kotlin. Please assign this to me as I have a deep knowledge in kotlin.
